include/itemlistformaction.h

```diff
@@ -106,29 +125,25 @@ class ItemListFormAction : public ListFormAction {
 
 	void prepare_set_filterpos();
 
-	void invalidate(const unsigned int invalidated_pos)
-	{
-		if (invalidation_mode == InvalidationMode::COMPLETE) {
-			return;
-		}
-
-		invalidation_mode = InvalidationMode::PARTIAL;
-		invalidated_itempos.push_back(invalidated_pos);
-	}
-
 	std::string item2formatted_line(const ItemPtrPosPair& item,
 		const unsigned int width,
 		const std::string& itemlist_format,
 		const std::string& datetime_format);
 
 	void goto_item(const std::string& title);
 
-	unsigned int pos;
+	void handle_op_saveall();
+
+protected:
```

Comment:
I understand the logic of putting methods and variables into separate `protected` sections, but the rest of Newsboat code doesn't use this style, so please don't do it here either.

Reply:
@Minoru done